functions/src/resolve-market.ts

```diff
@@ -74,6 +78,20 @@ export const resolvemarket = newEndpoint(opts, async (req, auth) => {
   const creator = await getUser(creatorId)
   if (!creator) throw new APIError(500, 'Creator not found')
 
+  if (contract.outcomeType === 'FREE_RESPONSE') {
```

Comment:
Wouldn't it make more sense for this whole block to be in `getResolutionParams`?